### PR TITLE
f-mfa@v0.10.0 - Added GTM Events (plus some minor fixes/tweaks)

### PR DESCRIPTION
**Pull request by oliversweb**
### Changed
- Changed mfa load success log message to `Info`.
- Help screen primary button text.
  
### Removed
- Mfa Form field validation/message.

### Added
- GTM events.
- Back button to error screen.

### CODE REVIEW

File: packages/components/pages/f-mfa/src/components/Help.vue [hunk 1]
```diff
@@ -29,7 +29,10 @@
                     <i18n
                         path="helpInfo.instructionsPoint4Text"
                         tag="li">
-                        <a :href="$t('helpInfo.instructionsHelpLink')"><strong>{{ $t('helpInfo.instructionsHelpText') }}</strong></a>
+                        <a
```
Comment (tommcclean) [inline packages/components/pages/f-mfa/src/components/Help.vue:32]: ❓- Will the instructions help link be another page in coreweb? If so could consider a `nuxt-link`

File: packages/components/pages/f-mfa/src/services/EventBuilder.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                }
+            };
+        case 's4':
+            return {
+                EventName: 'form_error',
+                Parameters: {
+                    component_name: 'multiFactorAuthentication',
+                    component_type: 'form',
+                    form_action: 'error',
+                    error_message: `${data}`
```
Comment (kevinrodrigues) [inline packages/components/pages/f-mfa/src/services/EventBuilder.js]: here you can just do `data`?

File: packages/components/pages/f-mfa/src/services/EventBuilder.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
+/* eslint-disable camelcase */
+// eslint-disable-next-line import/prefer-default-export
+export const buildEvent = (key, data = '') => {
+    switch (key) {
+        case 's1b':
```
Comment (tommcclean) [inline packages/components/pages/f-mfa/src/services/EventBuilder.js]: What are these keys? Did we make them up - could the meanings / names be more obvious to somebody not involved?

**Review by kevinrodrigues — DISMISSED** [review]

File: packages/components/pages/f-mfa/src/services/EventBuilder.js (hunk 3 above)
Comment (tommcclean) [inline packages/components/pages/f-mfa/src/services/EventBuilder.js]: ℹ️ - Could import keys from constants for consistency and less magic strings

Comment (kevinrodrigues) [reply packages/components/pages/f-mfa/src/services/EventBuilder.js]: I was going to suggest adding to a constant file with the key describing what it is and the value as `s1b`

File: packages/components/pages/f-mfa/src/components/Help.vue (hunk 1 above)
Comment (oliversweb) [reply packages/components/pages/f-mfa/src/components/Help.vue:32]: No, I think it goes to consumer help

File: packages/components/pages/f-mfa/src/services/EventBuilder.js (hunk 2 above)
Comment (xander-marjoram) [reply packages/components/pages/f-mfa/src/services/EventBuilder.js]: The only difference is that this does force it to be a string, even if someone were to pass an object in. TypeScript would save us here!

File: packages/components/pages/f-mfa/src/components/Help.vue (hunk 1 above)
Comment (xander-marjoram) [reply packages/components/pages/f-mfa/src/components/Help.vue:32]: Nope, it'll be linking to a help centre article

**Review by kevinrodrigues — DISMISSED** [review]

**Review by tommcclean — DISMISSED** [review]

**Review by tommcclean — DISMISSED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Review by tommcclean — APPROVED** [review]

**Pull request merged**